Add info about metrics endpoint and prometheus to documentation (#1543)
514-labs/moose: apps/framework-docs/src/pages/deploying/monitoring.mdx

# Monitoring your App

> Once you have deployed your MooseJS app you can visit `localhost:4000/metrics` to see the metrics that your moose app is collecting.

Reviewer: ```suggestion
When you run MooseJS app localy you can visit `localhost:4000/metrics` to see the metrics that your moose app is collecting. This endpoint is also available on your production docker container
```

> These metrics are collected using the [Prometheus](https://prometheus.io/) monitoring tool.

Reviewer: ```suggestion
These metrics are collected using [Prometheus](https://prometheus.io/) instrumentation and metrics format.
```
